app/models/user.rb

```diff
@@ -52,12 +52,20 @@ def set_default_role
     self.role ||= :user
   end
 
+  def confirm_piv_cac(proposed_uuid)
+    proposed_uuid && x509_dn_uuid == proposed_uuid
+  end
```

Comment:
Code Climate says this is not tested.

Reply:
What do you think about ending this method with a question mark since it is returning a boolean? Also, the order of the conditional confused me at first. What do you think about adding parentheses, such as:
```ruby
proposed_uuid && (x509_dn_uuid == proposed_uuid)
```
or maybe an early return:
```ruby
return false if proposed_uuid.blank?
x509_dn_uuid == proposed_uuid
```
I would want to make sure we test this with various types of empty values, like `nil`, an empty string, and whatever else could be passed in to try to make the conditional return true.

Reply:
perhaps `proposed_uuid && (proposed_uuid == x509_dn_uuid)` or `proposed_uuid == x509_dn_uuid if proposed_uuid`?

Reply:
Agree on testing with various false values: `''` and `nil` at least.